### PR TITLE
Update typescript to 4.9.3 and @types/node

### PR DESCRIPTION
**Pull request by KillariDev**

### CODE REVIEW

File: extension/inpage/package.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
 	"type": "module",
 	"devDependencies": {
-		"@types/node": "16.11.7",
+		"@types/node": "18.11.12",
```
Comment (MicahZoltu) [inline extension/inpage/package.json:4]:
```suggestion
```
`inpage` shouldn't have a dependency on `@types/node` as it is a browser targeted project.

File: extension/package.json [hunk 2]
```diff
@@ -23,10 +23,9 @@
 	},
 	"devDependencies": {
 		"@types/firefox-webext-browser": "94.0.1",
-		"@types/node": "16.11.40",
+		"@types/node": "18.11.12",
```
Comment (MicahZoltu) [inline extension/package.json:26]:
```suggestion
```
This project shouldn't have a dependency on this as it is a web targeted project.

**Review by MicahZoltu — APPROVED** [review]
@types/node should be removed from two of the projects, but can be in a separate PR.

File: extension/inpage/package.json (hunk 1 above)
Comment (KillariDev) [reply extension/inpage/package.json:4]: It's node project that compiles JavaScript files for the main project

Comment (MicahZoltu) [reply extension/inpage/package.json:4]: Oh, I see.  Ignore this one then.  Still should remove `@node/types` from the extension itself though.

**Pull request merged**